substrate/frame/support/procedural/src/derive_impl.rs

```diff
@@ -172,6 +172,31 @@ fn combine_impls(
 	final_impl
 }
 
+fn compute_disambiguation_path(
```

Comment:
Some internal docs please.

Reply:
Added in the code but will add here as well